packages/analytics/amplify_analytics_pinpoint_dart/test/analytics_client.dart

```diff
@@ -0,0 +1,114 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Unsure if this test file is really needed.  Since different storage instances are used there shouldn't really be any chance for data to intermingle.